src/Storefront/Resources/views/storefront/base.html.twig

```diff
@@ -35,8 +35,9 @@
             </noscript>
         {% endblock %}
 
+        {% set headerParameters = [] %}
```

Comment:
this means the "only" extension point here is to override the `base_esi_header` add stuff to the variable and then call parent? 

do we want to offer that extension point over PHP as well? E.g. have the header param and footer available in the storefront render event as structs where you can add scalar values and then make it globally available in twig and not just in this block

Reply:
done, please have a look again